### ISSUE TITLE
Review annotations to GO:0031992 energy transducer activity

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#2054]
Hello, 

EcoCyc has 4 proteins annotated to GO:0031992 energy transducer activity. The annotations are to an ABC-transporter, a function that is "energy dependent", but not energy "transducing" (ie, the energy is not converted from one form to another). 

See PMC4420915 "In the case of the ferrichrome receptor FhuA of Escherichia coli, binding induces unwinding of a short α-helix, resulting in a disordered Ton box in the periplasm (10,–12). Substrate passage into the periplasm may occur by conformational rearrangement of the plug domain following mechanical interactions with the Ton box (10, 13). These interactions require input of energy that is generated at the cytoplasmic membrane (CM) in the form of the proton motive force (PMF) (14, 15), harnessed and coupled to OM transport by the CM-embedded complex of ExbB, ExbD, and TonB (16). The Ton system is an attractive target for development of therapeutics due to its central position in powering active nutrient import in all Gram-negative bacteria."

I think the correct term might be 'proton motive force dependent protein transmembrane transporter activity' 

UniProtKB:P0ABV2
UniProtKB:P0ABU7
EcoGene:EG11012
UniProtKB:P02929

Can you please have a look ? 

Thanks, Pascale

**Comment by amandamackie** [comment]
Hi Pascale,

Ingrid passed this query on to me.

The proteins referenced are ExbB (P0ABU7), ExbD (P0ABV2) and TonB (P02929/EG11012). In gram negs, these proteins form a complex (the ‘Ton complex’) which is located in the inner membrane and extends across the periplasm to connect with outer membrane channels. The purpose of the complex is to transduce the energy of the proton motive force from the inner membrane to power import of iron-loaded siderophores and a few other compounds across the outer membrane. The best reference I can provide for this is a recent review PMID 26787763 [https://www.ncbi.nlm.nih.gov/pubmed/?term=26787763]. Another good one is PMID 27654919 [https://www.ncbi.nlm.nih.gov/pubmed/27654919] 

I don’t understand why you state that the annotations are to an ABC transporter – there is no involvement of ATP hydrolysis in the transport of iron-loaded siderophores across the outer membrane in E. coli. There are separate complexes in E. coli (eg. FepCDG; P23878, P23876, P23877) that function to transport iron siderophore complexes across the inner membrane, these complexes are powered by ATP hydrolysis but they act downstream of  Ton-mediated import across the the outer membrane. 

To summarise, the Ton complex energizes the transport of iron-loaded siderophores across the outer membrane, it does this by converting the PMF to kinetic energy which drives conformational change of the outer membrane channels. Once in the periplasm the iron siderophores are transported across the inner membrane by ABC transporters.

Regarding GO terms, I do not think the Ton complex proteins (TonB, ExbB and ExbD) should be annotated to PMF dependent transmembrane transporter activity; they could be given a BP term for iron transport but the best MF term is, in my opinion, ‘energy transducer activity’.

Amanda

**Comment by pgaudet** [comment]
Thank you Amanda, this is very useful. I got mislead thinking it's an ABC transporter based on this figure:
<img width="639" alt="image" src="https://user-images.githubusercontent.com/4782928/44140867-b4c86652-a07b-11e8-9678-1e428f6fd3e0.png">


I'll close this. Thanks, Pascale
